features/202.ngsild.IoTAgentUltralight.feature

```diff
@@ -0,0 +1,207 @@
+Feature: test tutorial 201.Introduction to IoT Sensors
+
+  This is the feature file of the FIWARE Step by Step tutorial for IoT Sensors - NGSI-LD
+  url: https://ngsi-ld-tutorials.readthedocs.io/en/latest/iot-agent.html
+  git-clone: https://github.com/FIWARE/tutorials.IoT-Agent.git
+  git-directory: /tmp/tutorials.IoT-Agent
+  # services orion | scorpio
+  shell-commands: git checkout NGSI-LD ; ./services create ; ./services ${CB_ENVIRONMENT:-orion}
+  clean-shell-commands: ./services stop
+
+
+  Background:
+    Given I set the tutorial 202 LD
+
+  Scenario: Checking the IoTAGent Service health
+    When  I send GET HTTP request to "http://localhost:4041/iot/about"
+    Then  I receive a HTTP "200" response code from IoTAgent with the body "response202-01.json" and exclusions "01.excludes"
+
+  Scenario Outline: Provisining a service Group
+    When  I prepare a POST HTTP request for "<description>" to "<url>"
+    And   I set header fiware-service to openiot
+    And   I set header fiware-servicepath to /
+    And   I set the body request as described in <file>
+    And   I perform the request
+    Then  I receive a HTTP response with status 201 and empty dict
+    Examples:
+      | url                                | file            | description         |
+      | http://localhost:4041/iot/services | 02.request.json | Provision a service |
+      | http://localhost:4041/iot/devices  | 03.request.json | Provision a device  |
+
+  Scenario: Sending some simulating it from dummy iot device - Request 4
```

Comment:
Maybe it is better to say:

Sending some simulation data from a dummy device to test the provision of the service group